shell/browser/native_window_views.cc

```diff
@@ -530,6 +530,20 @@ void NativeWindowViews::Unmaximize() {
 }
 
 bool NativeWindowViews::IsMaximized() {
+  // For window without WS_THICKFRAME style, we can not call IsMaximized().
+  // This path will be used for transparent windows as well.
+
+#if defined(OS_WIN)
+  gfx::Rect current_bounds = GetBounds();
+  if (!(::GetWindowLong(GetAcceleratedWidget(), GWL_STYLE) & WS_THICKFRAME)) {
+    // Compare the size of the window with the size of the display
+    auto display =
+        display::Screen::GetScreen()->GetDisplayNearestPoint(GetPosition());
```

Comment:
Should use `GetDisplayNearestWindow` to get window's display directly.

Reply:
Ok, sounds good. I switched to using that method with this next commit. Thanks

Reply:
@mlaurencin I think @zcbenz was saying to use GetDisplayNearestWindow here but maybe I'm missing something?

Reply:
@jkleinsc thank you for catching that. I changed it in in another function instead of this one. It should be correctly updated in the next commit.